### PR TITLE
Adding Gauge Card

### PR DESCRIPTION
**Pull request by zsarnett**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant-polymer) (if applicable):** home-assistant/home-assistant-polymer#1742

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_lovelace/guage.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+sidebar_label: Gauge
+description: "The Gauge card allows you to display sensor information visually"
+date: 2018-10-11 10:28 +00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+Gauge card is a basic card that allows visually seeing sensor data
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: Sentence should end with a full stop (`.`)

File: source/_lovelace/guage.markdown [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  type: string
+unit_of_measurement:
+  required: false
+  description: Unit of Measurement given to data
+  type: string
+  default: "Unit Of Measurement given by entity"
+min:
+  required: false
+  description: Minimum value for graph
+  type: number
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `number` is an invalid data type for the documentation, change to `integer`.

File: source/_lovelace/guage.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  default: 0
+max:
+  required: false
+  description: Maximum value for graph
+  type: number
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `number` is an invalid data type for the documentation, change to `integer`.

File: source/_lovelace/guage.markdown [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## {% linkable_title Severity Variables %}
+
+If you define the severity object, these are the required options:
+
+{% configuration %}
+green:
+  required: true
+  description: Value from which to start green color
+  type: number
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `number` is an invalid data type for the documentation, change to `integer`.

File: source/_lovelace/guage.markdown [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+yellow:
+  required: true
+  description: Value from which to start yellow color
+  type: number
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `number` is an invalid data type for the documentation, change to `integer`.

File: source/_lovelace/guage.markdown [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+red:
+  required: true
+  description: Value from which to start red color
+  type: number
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `number` is an invalid data type for the documentation, change to `integer`.

File: source/_lovelace/guage.markdown [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+{% endconfiguration %}
+
+## {% linkable_title Examples %}
+
+Basic example:
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: The basic example should be placed above the configuration variables list and should only contain the required parameters.

File: source/_lovelace/guage.markdown [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+  default: 0
+max:
+  required: false
+  description: Maximum value for graph
+  type: number
+  default: 100
+severity:
+  required: false
+  description: Allows setting of colors for different numbers
+  type: object
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `object` is not a valid documentation type, you probably need `map` here.

File: source/_lovelace/guage.markdown [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+{% endconfiguration %}
+
+## {% linkable_title Severity Variables %}
+
+If you define the severity object, these are the required options:
+
+{% configuration %}
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: These are part of the `severity` configuration option and should be defined as `keys:` under that `map`.

File: source/_lovelace/guage.markdown [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+  unit_of_measurement: '%'
+  entity: sensor.cpu_usuage
+```
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_gauge_card.gif' alt='Screenshot of the gauge card with custom title and unit of measurement'>
+Screenshot of the gauge card with custom title and unit of measurement.
+</p>
+
+Define severity object:
```
Comment (frenck) [inline source/_lovelace/guage.markdown]: `the` missing: `Define the severity object`

**Review by frenck — CHANGES_REQUESTED** [review]
@zsarnett Thanks for the PR!
I've reviewed it and left you some comments. Could you please take a look? 👍

File: source/_lovelace/guage.markdown (hunk 8 above)
Comment (zsarnett) [reply source/_lovelace/guage.markdown]: https://www.home-assistant.io/lovelace/picture-glance/#state_image

I was referencing this variable. I will change it to map though if that is the better type

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating @zsarnett! Looks great 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**